exonum/src/storage/base_index.rs

```diff
@@ -112,7 +112,7 @@ where
     /// Returns a value of *any* type corresponding to the key of *any* type.
     pub fn get<K, V>(&self, key: &K) -> Option<V>
     where
-        K: StorageKey,
+        K: StorageKey + ?Sized,
```

Comment:
If you define `StorageKey` as 

```
pub trait StorageKey: ToOwned + ?Sized
```

so you don't need to add `+ ?Sized` every time

Reply:
Unfortunately:
```rust
error: `?Trait` is not permitted in supertraits
  --> exonum/src/storage/keys.rs:62:34
   |
62 | pub trait StorageKey: ToOwned + ?Sized {
   |                                  ^^^^^
   |
   = note: traits are `?Sized` by default
```